src/components/ListView/index.js

```diff
@@ -3,7 +3,8 @@ import ListViewDataSource from './ListViewDataSource';
 import ListViewPropTypes from './ListViewPropTypes';
 import ScrollView from '../ScrollView';
 import StaticRenderer from '../StaticRenderer';
-import React, { Component, isEmpty, merge } from 'react';
+import React, { Component } from 'react';
+import { isEmpty, merge } from 'lodash';
```

Comment:
If you look over the commit history you'll see I removed lodash in favor of platform features and smaller alternatives